src/abi/Registry.json

```diff
@@ -1,5 +1,5 @@
 {
-  "address": "0x4F9C569AfD5F3F0Bfa3bF2c7443544581330459e",
+  "address": "0x4a84e406ED2054bA24244dfE8660F055aC00b971",
```

Comment:
i dont think we need to include these addresses anymore as they're fetched from the config - just the abi's will do imo

Reply:
Agree, leaving them in causes confusion.